src/react-count-to.js

```diff
@@ -13,48 +12,59 @@ const CountTo = React.createClass({
     className: React.PropTypes.string,
   },
 
+  getDefaultProps() {
+    return {
+      from: 0,
+      delay: 100,
+      digits: 0,
+      onComplete: () => {},
```

Comment:
I would prefer to set the default props only for props that have a default value (`from`, `delay` and `digits`).
Can you please remove `onComplete` from the list?